apps/gro/steps.js

```diff
@@ -8,6 +8,7 @@ module.exports = {
     next: '/about'
   },
   '/about': {
+    template: 'step',
```

Comment:
shame we need to tell it the template is called "step", when it should possibly be the default?